makeflow/src/makeflow.c

```diff
@@ -1578,6 +1584,7 @@ int main(int argc, char *argv[])
 	batch_queue_set_option(remote_queue, "caching", cache_mode ? "yes" : "no");
 	batch_queue_set_option(remote_queue, "wait-queue-size", wq_wait_queue_size);
 	batch_queue_set_option(remote_queue, "working-dir", working_dir);
+	batch_queue_set_option(remote_queue, "master-prefer-hostname", master_prefer_hostname);
```

Comment:
Do you suggest I then pass yes or no on to the Catalog or equate those to values (1 or 0)  in batch_job_work_queue.c?